snuba/migrations/operations.py

```diff
@@ -53,34 +54,21 @@ def format_sql(self) -> str:
         return f"DROP TABLE IF EXISTS {self.table_name};"
 
 
-class TableEngine:
-    pass
-
-
-class ReplacingMergeTree(TableEngine):
-    def __init__(self, version_column: str):
-        self.__version_column = version_column
-
-    def __str__(self) -> str:
-        return f"ReplacingMergeTree({self.__version_column})"
-
-
 class CreateTable(SqlOperation):
     def __init__(
         self,
         storage_set: StorageSetKey,
         table_name: str,
         columns: Sequence[Column],
         engine: TableEngine,
-        order_by: str,
     ):
         self.__table_name = table_name
         self.__columns = columns
         self.__engine = engine
-        self.__order_by = order_by
         super().__init__(storage_set)
 
     def format_sql(self) -> str:
         columns = ", ".join([col.for_schema() for col in self.__columns])
+        engine = self.__engine.get_create_table_statement()
 
-        return f"CREATE TABLE IF NOT EXISTS {self.__table_name} ({columns}) ENGINE {self.__engine} ORDER BY {self.__order_by};"
+        return f"CREATE TABLE IF NOT EXISTS {self.__table_name} ({columns}) ENGINE {engine};"
```

Comment:
I think it would be wise to clarify which parts of the migration framework is storage specific and which part (if any) is not. I think now (and probably for a long time) everything is storage specific, so it would be useful to state it explicitly somewhere. Probably in the migration class and in the operation classes. So none would think about adding logical migration concepts or anything entity related to the framework.

Reply:
Yeah, the way it was earlier was a mistake. Orderby is only relevant for merge tree and merge engines. Everything is storage specific.

Reply:
I'm not sure this comment is relevant to the overall migration class and the runner which are now very generic and don't reference or assume we're using operations at all (except multi step migration). This comment is also only relevant to the create table operation and not operations more generally.